artifacts/contracts/discrepancies/nonce/InternalCallee.sol/Sample.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Question: How comes there's this Sample.json artifact file but no Sample.sol smart contract file?

Reply:
The `Sample` contract is defined [here](https://github.com/hashgraph/hedera-smart-contracts/pull/654/files#diff-3288fef155ccad3900a0de8a58fd259640030de83d579ab7c630782a5b1959a5R4) and as it's a standalone contract, separate artifact should be created as well.